etest/qemu.py

```diff
@@ -0,0 +1,28 @@
+"""QEMU interpreter management."""
+# Copyright (C) 2014 by Alex Brandt <alunduil@alunduil.com>
+#
+# etest is freely distributable under the terms of an MIT-style license.
+# See COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+from etest import docker
+
+
+def start():
+    """Start the QEMU interpreter."""
+    docker.pull("multiarch/qemu-user-static")
+
+    global QEMU_CONTAINER
```

Comment:
We should avoid globals.  If we make this a context manager then we could easily inject this value via the context.

Reply:
Thats true. I will look into making it a context manager.